renesas: g2l: Add support ATF bootup through FCONF framework device tree

ThanhNguyenRVC opened the pull request:
Support ATF bootup through FCONF framework device tree for RZ/G2L-SBC and RZ/G2L-EVK.

Changes include:
- Modify TF-A with FCONF configuration for RZ/G2L-SBC, RZ/G2L-EVK
- Add an essential node (CPG, SYSC, PFC, DDR, SPI) for booting through FCONF configuration.

vudangRVC commented on fdts/rzpi.dts:
```diff
@@ -0,0 +1,163 @@
+			#address-cells = <1>;
+			#size-cells = <0>;
+			status = "disabled";
+			spim_phycnt = <0x260>;
```
@ThanhNguyenRVC, 
I think most values of the fields below this comment can be defined in a header instead of being hardcoded.
To know how to define them, check the hardware to understand the settings value. Then define a proper name.
Additionally, not all the fields are necessary. We can remove them, please think about it. But let's re-check when we support other platforms first.

The above comments are for both device trees.

vudangRVC commented:
Looks good to me as a first PR for FCONF support. Thanks

ThanhNguyenRVC committed:
renesas: g2l: Add support ATF bootup through FCONF framework device tree

Change includes:
	- Support ATF bootup through FCONF framework device tree for RZ/G2L-SBC and RZ/G2L-EVK.

Signed-off-by: Thanh Nguyen <thanh.nguyen.wx@bp.renesas.com>

ThanhNguyenRVC committed:
renesas: g2l: sbc: Update device name
